src/tools/wasm-emscripten-finalize.cpp

```diff
@@ -254,6 +255,15 @@ int main(int argc, const char* argv[]) {
     }
   }
 
+  // Convert the imports to indirects before the Legalization pass to 
+  // avoid unnecessary legalization
+  if (!options.passOptions.arguments["library-file"].empty()) {
```

Comment:
Can you give this argument a better name too?   How about "js-symbols"?

Reply:
Sure, sounds good.